tests/test_aircon.py

```diff
@@ -71,8 +72,10 @@ async def test_attributes(
     mock_appliance_http_get(
         appliance_http_client_mock, backend_selector_mock, SAID, DATA1
     )
-
-    aircon = Aircon(backend_selector_mock, auth_mock, SAID)
+    appliance_http_client_mock.create_session(asyncio.get_event_loop())
```

Comment:
since the session is created on the test now, at the end of the test we should close the session, otherwise we get "Unclosed client session" warnings.
Same for the other tests

Reply:
That is weird because I got them locally. Maybe it is because you are using windows or something like that.